auxx/src/Command/Proc.hs

```diff
@@ -113,6 +116,7 @@ createCommandProcs mpm hasAuxxMode printAction mDiffusion = rights . fix $ \comm
     },
 
     let name = "addr-hd" in
+    needsProtocolConstants name >>= \pc ->
```

Comment:
is someone afraid of do notation? :thinking:

Reply:
Good point - I might leave this for another PR though, because this whole function is a mess!